heimdall2/values.yaml

```diff
@@ -64,25 +112,16 @@ heimdall:
   image:
     repository: mitre/heimdall2
     pullPolicy: IfNotPresent
-    tag: "release-latest"
+    tag: "2.8.7"
```

Comment:
I think the default should be release-latest. Individuals running this chart can easily pass the tag as a different value if they want to pin to a particular release or update release by release.

Reply:
Talked about this on slack/voice.

Reverting this for now.  Plan is to add automation to auto-update/release the chart on Heimdall releases.